tests/components/meteo_france/test_weather.py

```diff
@@ -0,0 +1,47 @@
+"""Test Météo France weather entity."""
+
+from homeassistant.components.meteo_france.const import CONF_CITY, DOMAIN
+from homeassistant.components.weather import (
+    ATTR_CONDITION_PARTLYCLOUDY,
+    ATTR_WEATHER_HUMIDITY,
+    ATTR_WEATHER_PRESSURE,
+    ATTR_WEATHER_TEMPERATURE,
+    ATTR_WEATHER_WIND_BEARING,
+    ATTR_WEATHER_WIND_GUST_SPEED,
+    ATTR_WEATHER_WIND_SPEED,
+)
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_weather(hass: HomeAssistant) -> None:
```

Comment:
I think this would be better making the using "snapshot" assertion, possibly with a new "mock_config_entry" fixture that can be reused in other tests

Reply:
I'm not sure I get what you mean, but I took inspiration from the Met.no test suite, and their [mock_weather fixture](https://github.com/home-assistant/core/blob/83b34c6fafec4fdc80b2c2180740dde56dc410e1/tests/components/met/conftest.py#L9) looks like what you ask.

Reply:
Sample fixture to create the config entry:
https://github.com/home-assistant/core/blob/83b34c6fafec4fdc80b2c2180740dde56dc410e1/tests/components/onewire/conftest.py#L34-L35

Sample use of snapshots (binary_sensor here, but the same applies to weather):
https://github.com/home-assistant/core/blob/83b34c6fafec4fdc80b2c2180740dde56dc410e1/tests/components/onewire/test_binary_sensor.py#L28-L35